src/Spectre.libStatistics/StatisticalIndex.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+* OrderableIndex.cpp
+* ENTER CLASS DESCRIPTION HERE.
```

Comment:
This PR is so meta, even the source gives you useful hints

Reply:
I've created file templates for VS, so these things became easy to catch :P